tests/transformer/test_operators.py

```diff
@@ -0,0 +1,282 @@
+from RestrictedPython._compat import IS_PY35_OR_GREATER
+from tests import c_eval
+
+import pytest
+
+
+# Arithmetic Operators
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_Add(c_eval):
+    result = c_eval('1 + 1')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_Sub(c_eval):
+    result = c_eval('2 - 1')  # why is this giving an exception?
+    # result = c_eval('2 + 1')
+    assert result.code is None
```

Comment:
I currently test against our implementation and would like to discuss which functionallity should be allowed, I think all of these should be allowed, but why the hell is the test failing.